src/style.css

```
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
```

```suggestion
  font-family: "Avenir-heavy";
```
rename this font and use new name in :focus